### DISCUSSION TITLE
Delete a running job in DPS

### DISCUSSION THREAD

**Discussion by emileten** [MAAP-Project/Community#835] [Platform] [answered]
Can users who don't have direct access to the DPS backend delete jobs they've submitted and that are still running ? If so, what is the procedure ?

I've been trying to run a simple test job that seems to have a long running loop bug problem, so although I know it's currently failing, it hasn't actually stopped, and I don't know when it will. In the meanwhile it's wasting resources so it would be great to have a quick way to kill that process. 

@wildintellect @sujen1412 @jjfrench ? Not sure who to tag.

**Comment by emileten** [comment]
Some slack browsing seems to indicate that usually people ask devs to kill jobs. Would it be possible to give more people access to the backend where this operation is done ? In particular the data team. In the meanwhile, here are the jobs I'd like to kill

- `7b1d2605-6078-4ca1-95e1-0c9ba13e147f`
- `e7d0a1bc-eaa1-4010-89eb-42ccc3b1ac18`

At the time this message will be read maybe those jobs will be done/dead already, but in case, I am posting these ids for someone with credentials to go and kill them.

**Reply by sujen1412** [reply]
The jobs were killed last night.

**Reply by wildintellect** [reply]
Does maap.dismissJob() or maap.deleteJob() not work? neither of these are documented in https://docs.maap-project.org/
but could potentially be covered if we deal with https://github.com/MAAP-Project/maap-py/issues/44 (autogeneration of api documentation)

**Reply by emileten** [reply]
I wasn't aware of `maap.dismissJob()` or `maap.deleteJob()`. Assuming these methods work I marked your message as answer. Yes we should document them.

**Comment by sujen1412** [comment, marked as answer]
Currently, the dismiss and deleteJob do not work. As we are trying to clean up the maap-py interface we will unify this in one function and document it. 
Follow https://github.com/MAAP-Project/Community/issues/640
